app/src/main/java/com/honeycake/tictactoe/ui/composable/ButtonItem.kt

```diff
@@ -21,6 +21,7 @@ import com.honeycake.tictactoe.ui.theme.Typography
 fun ButtonItem(
     modifier: Modifier = Modifier,
     text: String,
+    isEnabled: Boolean,
```

Comment:
make the default value true so you did not need to call isEnabled = true every line needed